user_docs/en/userGuide.md

```diff
@@ -1831,6 +1831,38 @@ This information greatly aides NV Access to prioritize future development of NVD
 If this is enabled, NVDA will inform you when there is a pending update on startup, offering you the possibility to install it.
 You can also manually install the pending update from the Exit NVDA dialog (if enabled),  from the NVDA menu, or when you perform a new check from the Help menu.
 
+##### Update Mirror {#UpdateMirror}
+
+These controls allow you to specify an alternative URL to use to check for updates to NVDA.
+This may be of use in locations where access to the NV Access NVDA update server is slow or unavailable.
```

Comment:
```suggestion
This may be of use in locations where access to the NV Access NVDA update server is slow or blocked.
```

Reply:
I avoided "blocked" as it feels a little politically charged. @Qchristensen, @seanbudd what do you think?

Reply:
My thinking is that users may wonder "Why would a URL be unavailable?" But "blocked" is pretty clear in its meaning.
As for politically charged: Users in countries without this problem aren't likely to find it so. I have never heard users in countries that do have it be so squeamish about terminology, but maybe I don't travel in the right circles. :)

Reply:
I think unavailable is more generic and useful here - for example it covers corporate firewalls and NV Access downtime. I think its fine either way, but I'm also not sure if blocked is quite accurate based on conversations I've had with @cary-rowen . It seems like access in PRC is a mixture of totally fine, throttling, not being part of an allowlist, and being part of a blocklist depending on your ISP.